Documentation/special-services/pod-mutator.md

```diff
@@ -0,0 +1,74 @@
+# PodMutator Service
```

Comment:
`Manager`isms are often too abstract, but here it fits quite well, I'm undecided which one is better.